morebits.css

```diff
@@ -186,10 +187,6 @@ div.morebits-previewbox .mw-editsection
 	background-image: none;
 }
 
-body.skin-monobook .morebits-dialog {
-	font-size: 125%;
-}
```

Comment:
This (at 0,2,1) was being over-ridden by the more specific `.ui-dialog.morebits-dialog .morebits-dialog-content` selector (0,3,0) so this wasn't doing anything at all.

Reply:
Well, it was doing *something*.  I get 11.44px with, 11px without.  More generally, shifts things around a bit.  Continued below...